integration/alertmanager_test.go

```diff
@@ -40,3 +48,56 @@ func TestAlertmanager(t *testing.T) {
 	// Ensure no service-specific metrics prefix is used by the wrong service.
 	assertServiceMetricsPrefixes(t, AlertManager, alertmanager)
 }
+
+func TestAlertmanagerStoreAPI(t *testing.T) {
```

Comment:
The last CI execution failed:
```
=== RUN   TestAlertmanagerStoreAPI
15:33:37 Starting minio-9000
15:33:39 Ports for container: e2e-cortex-test-minio-9000 Mapping: map[9000:32772]
15:33:39 minio-9000: Attempting encryption of all config, IAM users and policies on MinIO backend
15:33:40 Starting alertmanager
15:33:41 Ports for container: e2e-cortex-test-alertmanager Mapping: map[80:32774 9095:32773]
    TestAlertmanagerStoreAPI: alertmanager_test.go:80: 
        	Error Trace:	alertmanager_test.go:80
        	Error:      	Received unexpected error:
        	            	setting config failed, 
        	Test:       	TestAlertmanagerStoreAPI
15:33:41 Killing alertmanager
15:33:41 Killing minio-9000
15:33:41 minio-9000: Exiting on signal: TERMINATED
--- FAIL: TestAlertmanagerStoreAPI (6.15s)
```

Details: https://app.circleci.com/pipelines/github/grafana/cortex/623/workflows/50bb6d2d-158a-494e-b251-0ade70938ce1/jobs/7906/steps

Could you take a look, please?